components/Layout.tsx

```diff
@@ -83,6 +87,16 @@ export default function Layout(props: LayoutProps) {
         href: "/signin",
         inner: "Sign in",
       },
+    props.session && props.notificationsCount! > 0
+      ? {
+        href: "/notifications",
+        inner: (
+          <span class={notificationsStyle}>
+            {props.notificationsCount}
```

Comment:
I'm not sure I got this here.
In GitHub, the blue dot is on the /notifications link, not on the accounts link.

In any case, if the dot was in the accounts link, what would happen if a user clicked on this link? would it lead the user to the /account page or the the /notifications page?
It could may be the case that the user clicks on this accounts button and a drop down menu appears where a user can select between the action /account page or the /notifications page. I like this approach, but I'd suggest we tackle this later as it seems more complex.

Reply:
Fair points. Let me refine my suggestion:
1. Have a notifications icon link in the header menu.
2. The link goes to `/account/notifications`.
3. When there are "unread" notifications, there is a dot on the link. When all notifications are "read", there is no dot on the link.

WDYT?

Reply:
I tried following your suggestion. Not sure if I liked the final design, but I guess we can prettify it later